pkg/csi_driver/multishare_controller.go

```diff
@@ -80,10 +80,13 @@ func (m *MultishareController) CreateVolume(ctx context.Context, req *csi.Create
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 
-	_, err = getShareRequestCapacity(req.GetCapacityRange())
+	reqBytes, err := getShareRequestCapacity(req.GetCapacityRange())
 	if err != nil {
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
+	if !util.IsAligned(reqBytes, util.Gb) {
+		return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("Invalid requested size(bytes) %d", reqBytes))
```

Comment:
consider add more info here: "size need to be multiply of 1GiB"

Reply:
done